RELEASE.md

```diff
@@ -0,0 +1,29 @@
+Release 1.2.1 of WebRTC C SDK
+
+Release tagged at: <Commit:  <to-be-filled>
+
+Whats’s new:
+- ICE server stats added
+- RTP outbound and inbound stats added (open source contribution)
+- Signaling stats available
+- Clang check added to Travis
+- MBedTLS support added. Choice between OpenSSL and mbedTLS available
```

Comment:
I think we should postpone this to the next release since I still need to verify some key metrics.

Reply:
Ok. But, the changes would still be part of the release. Do we want to add a note, maybe something like MBedTLS (beta version). ?

Reply:
@disa6302 That's a good idea. I like it.